vxblastsms/tests/test_ussd.py

```diff
@@ -48,28 +44,33 @@ def assert_inbound_message(self, msg, **field_values):
         for field, expected_value in expected_field_values.iteritems():
             self.assertEqual(msg[field], expected_value)
 
-    def assert_outbound_message(self, msg, content, callback,
+    def assert_outbound_message(self, outbound_msg, appid, config_app_id,
+                                sessionid, reply_content, msisdn,
                                 continue_session=True):
-        headertext = '<headertext>%s</headertext>' % content
+        se_msisdn = '<msisdn>%s</msisdn>' % str(msisdn)
+        se_sessionid = '<sessionid>%s</sessionid>' % str(sessionid)
+        se_msg = '<msg>%s</msg>' % str(reply_content)
```

Comment:
I followed the AAT example of doing that, was a bit strange to me too :)  Opened issue #5 for addressing this.  Discussed with @imsickofmaps - will implement once we've seen what the live system does in testing.